src/test/java/com/commercetools/sync/commons/MockUtils.java

```diff
@@ -22,6 +40,91 @@ public static CustomFieldsDraft getMockCustomFieldsDraft() {
         customFieldsJsons.put("invisibleInShop", JsonNodeFactory.instance.booleanNode(false));
         customFieldsJsons
             .put("backgroundColor", JsonNodeFactory.instance.objectNode().put("de", "rot").put("en", "red"));
-        return CustomFieldsDraft.ofTypeKeyAndJson("StepCategoryTypeKey", customFieldsJsons);
+        return CustomFieldsDraft.ofTypeIdAndJson("StepCategoryTypeId", customFieldsJsons);
+    }
+
+    /**
+     * Creates a mock instance of {@link CustomFieldsDraft} with the supplied {@code customTypeId} and
+     * {@code customFields}.
+     *
+     * @return a mock instance of {@link CustomFieldsDraft} with supplied id and custom fields.
+     */
+    public static CustomFieldsDraft getMockCustomFieldsDraft(@Nullable final String customTypeId,
+                                                             @Nonnull final Map<String, JsonNode> customFields) {
+        final CustomFieldsDraft newCategoryCustomFieldsDraft = mock(CustomFieldsDraft.class);
```

Comment:
Wouldn't be better to test with real draft objects? Unless you verify method calls.